tests/baselines/reference/user/puppeteer.log

```diff
@@ -1,6 +1,9 @@
 Exit Code: 2
 Standard output:
 src/common/Target.ts(95,21): error TS2322: Type '(value: boolean | PromiseLike<boolean>) => void' is not assignable to type '() => void'.
+src/common/fetch.ts(21,3): error TS2322: Type '((input: RequestInfo, init?: RequestInit) => Promise<Response>) | typeof import("../../../node_modules/@types/node-fetch/index")' is not assignable to type '(input: RequestInfo, init?: RequestInit) => Promise<Response>'.
+  Type 'typeof import("../../../node_modules/@types/node-fetch/index")' is not assignable to type '(input: RequestInfo, init?: RequestInit) => Promise<Response>'.
+    Type 'typeof import("../../../node_modules/@types/node-fetch/index")' provides no match for the signature '(input: RequestInfo, init?: RequestInit): Promise<Response>'.
```

Comment:
node-fetch isn't compatible with fetch anymore, but I can't figure out why. However, it's the same on 4.1 and 4.0, so I'm going to merge for now.